torch/onnx/symbolic_opset9.py

```diff
@@ -2201,8 +2202,9 @@ def batch_norm(
         return res
 
 
-@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
-def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+def _layer_norm_shared(
```

Comment:
Consider a more descriptive name for the function?

Reply:
Ok.